tests/PySys/tedge_connect_test_positive/run.py

```diff
@@ -25,7 +25,7 @@ def execute(self):
             stdouterr="tedge_connect_c8y_test_positive",
         )
         self.device_fragment = self.cumulocity.get_thin_edge_device_by_name(
-            self.project.deviceid)
+            self.project.device)
```

Comment:
In pysys, the `deviceid` variable does not refer to the Thin Edge `device id` but rather the numeric Cumulocity `device id`. This variable `device` refers to the Cumulocity device name which is the Thin Edge `device id`.